presentations/2019-08-21-go-present/README.md

```diff
@@ -1,2 +1,5 @@
 # 2019-08-21 Go Present - Adam Schwartz
 Presentation slides can be found here: [https://github.com/anschwa/talks/tree/master/present](https://github.com/anschwa/talks/tree/master/present)
+
+# 2019-08-21 String Matching - Matt McQuillan
+Presentation slides can be found here: [Google docs slideshow](https://docs.google.com/presentation/d/10je2jFppqy-n1G4epebPRIpKjdvaYvMp4ADmC1Lgm5M/edit?usp=sharing)
```

Comment:
This should be in its own folder and file: `../presentations/2019-08-21-string-matching/README.md`